clang/lib/CodeGen/CGCall.cpp

```diff
@@ -4681,6 +4676,12 @@ void CallArg::copyInto(CodeGenFunction &CGF, Address Addr) const {
   IsUsed = true;
 }
 
+void CodeGenFunction::EmitWritebacks(CodeGenFunction &CGF,
+                                     const CallArgList &args) {
+  for (const auto &I : args.writebacks())
+    emitWriteback(CGF, I);
+}
```

Comment:
Since this requires a reference to a CodeGenFunction, why not just make this a regular method instead of a static method?

In fact, after double-checking, it appears that it is a regular method, so why does it need a separate input argument when it already has `this`?

```suggestion
void CodeGenFunction::EmitWritebacks(const CallArgList &args) {
  for (const auto &I : args.writebacks())
    emitWriteback(*this, I);
}
```